### DISCUSSION TITLE
How to use Organic Maps and Google Earth together (and keep list names, bookmark colors...)

### DISCUSSION THREAD

**Discussion by nonobio** [organicmaps/organicmaps#8204] [Q&A]
Hi,

Following [this reply on an issue](https://github.com/organicmaps/organicmaps/issues/8201#issuecomment-2116946931), it's interesting to know that the kmz is a simple zip, which allows you to extract the kml files separately if necessary.

I tried to edit the kmz OM file opened in Google Earth Windows, I then saved the location in kmz, but it does not open in OM.

Working on kml files seems to be less prone to errors but I often have problems with kml file names and/or the presence of an unnecessary subfolder.

For now, this is how I do it:
- Export all lists at once with Organic Maps
- Unzip the files
- Open the kml files located in files in Google Earth Windows
- Following modifications in Google Earth, save the list in kml
- If renaming the name of the list, open the kml saved by Earth in a text editor (Notepad++ for example) and also rename the following line:
`<ExtendedData>
		<mwm:name>
      <mwm:lang code="default">NEWNAME</mwm:lang>`
- Open the file again in Google Earth and if there is an extra subfolder, delete it

The file can then be opened in Organic Maps and so on...

On the other hand, I can't always keep the colors of the bookmarks. For example if I create a bookmark in Google Earth, choose custom icon and put the address OrganicMaps, for instance https://omaps.app/placemarks/placemark-green.png or https://omaps.app/placemarks/placemark-red.png, the icons have the correct colors in Earth, but once the file is saved in kml and opened in Organic Maps, I end up with red colors only.
Opening the kml in an editor, I see duplicates of these lines:
`<Style id="placemark-red1">`
`<Style id="placemark-red0">`
I think it comes from there...

In short, I'm pretty much getting by but it's not easy. Do you have any tips for simplifying this and keeping the bookmark colors between Earth and Organic?

Thanks :)

**Comment by biodranik** [comment]
See #2795 for custom KMZ icons support

**Reply by nonobio** [reply]
Thanks @biodranik 
If I understood correctly, this PR should solve my icons problem?
Should I wait for this to be added to a future release?
(I'm not a programmer)
